kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flight_finder/Journey.java

```diff
@@ -0,0 +1,7 @@
+package com.kodilla.good.patterns.flight_finder;
+
+public interface Journey
```

Comment:
czym się różni ten interfejs od Flight? poza tym, że metody nazywają się inaczej? 
Chyba wiem jakie miałeś intencje, aczkolwiek ja bym tutaj dodał jeszcze metodę, która zwraca kolekcję lotów tak, żeby umożliwić zebranie listy lotów, z których składa się podróż... co Ty na to? 

Wtedy też będziesz mógł zmienić getConnectingFlights() tak, żeby zamiast listy Stringów zwracane były obiekty, na których można dalej pracować w kodzie. Stringi są mniej użyteczne, prawda ?

Reply:
Gruntownie to przebudowałem zgodnie z sugestią, tak, że teraz wyszukiwarka robi dobrą robotę dla miłośników latania:). I poszło na to prawie 3 dni:(. Mam problem z sensownym debugowaniem kodu (step by step). Ale raczej działa poprawnie.